packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -1,16 +1,192 @@
 import { IDesignSystem } from "../design-system";
-import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
 import { IButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { IMSFTButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyType } from "../utilities/typography";
+import * as Chroma from "chroma-js";
+import { isNullOrUndefined } from "util";
 
-const styles: ComponentStyles<IButtonClassNameContract, IDesignSystem> = {
+function applyTransaprentBackplateStyles(): ICSSRules<IDesignSystem> {
+    return {
+        extend: "button",
+        backgroundColor: (): string => {

```

Comment:
This exists in a few places. Can we remove duplication by creating a separate function?

Reply:
You'll have to be specific as I dont know what code you mean because github doesnt select.

Reply:
background color as transparent.